src/EditorFeatures/CSharpTest/Structure/ArgumentListSyntaxStructureTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Structure;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Structure;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Structure;
+
+[Trait(Traits.Feature, Traits.Features.Outlining)]
+public class ArgumentListSyntaxStructureTests : AbstractCSharpSyntaxNodeStructureTests<ArgumentListSyntax>
+{
+    internal override AbstractSyntaxStructureProvider CreateProvider() => new ArgumentListStructureProvider();
+
+    [Fact]
+    public async Task TestInvocationExpressionSingleLine()
+    {
+        var code = """
+            var x = M$$();
+            """;
+
+        await VerifyBlockSpansAsync(code);
+    }
+
+    [Fact]
+    public async Task TestInvocationExpressionThreeLines()
+    {
+        var code = """
+            var x = M$${|span:(
+                "",
+                "")|};
```

Comment:
what's the experience you have in the IDE if you have:

```c#
var v = M(N(
   "",
   ""));
```

In other words, how does nesting impact things, when the outer/inner are on the same start/end lines.

Reply:
@CyrusNajmabadi I wasn't able to get RoslynDeployment work properly. https://www.youtube.com/watch?v=ND8dDO6oJ4U see around 38:00